code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -490,6 +491,29 @@
 				if(G.summoner)
 					G.summoner << "<span class='danger'><B>[AM] has crossed your surveillance trap at [get_area(snare_loc)].</span></B>"
 
+/mob/living/simple_animal/hostile/guardian/ranged/Shoot(atom/targeted_atom)
+	var/obj/item/projectile/guardian/shard = ..()
+	if(!shard)
+		return
+	shard.color = guardiancolor
+
+/proc/techcolor2hex(var/colour)
```

Comment:
Colours like neon aren't defined in color2hex.
The snowflake proc is to choose colors best suited for the guardians as John stated.